gulpfile.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+var gulp = require('gulp'),
+    less = require('gulp-less'),
+    exec = require('gulp-exec'),
+    nodemon = require('gulp-nodemon'),
+
+    sequence = require('run-sequence'),
+    source = require('vinyl-source-stream'),
+    browserify = require('browserify'),
+    del = require('del');
+
+var options = {
+        continueOnError: true,
+        pipeStdout: true
+    };
+
+gulp.task('clean', del.bind(null, ['client']));
+
+gulp.task('styles', function () {
```

Comment:
Regarding gulp in dependencies: Would rather do that because it bloats the installation size.